conformity/fields/net.py

```diff
@@ -117,15 +122,19 @@ def expand_ipv6_address(value):
             ret_ip.append(('0' * (4 - len(hextet)) + hextet).lower())
         return ':'.join(ret_ip)
 
-    def introspect(self):
-        return strip_none({
+    def introspect(self, include_conformity_type=False):
+        result = strip_none({
             "type": self.introspect_type,
             "description": self.description,
         })
+        if include_conformity_type:
+            result["conformity_type"] = self.conformity_type
+        return result
 
 
 IPAddress = functools.partial(
     Any,
     IPv4Address(),
     IPv6Address(),
+    conformity_type="ip_address",
```

Comment:
As all the args & kwargs are sent to `Any`'s constructor, the only way I can think of keeping the partial object but returning the appropriate `conformity_type`(`type` is `any`)